Sources/PointFree/Account/Account.swift

```diff
@@ -214,37 +214,61 @@ private let titleRowView = View<Prelude.Unit> { _ in
     ])
 }
 
-private let profileRowView = View<(Database.User, [Database.EmailSetting])> { currentUser, currentEmailSettings in
-  gridRow([`class`([Class.padding([.mobile: [.bottom: 4]])])], [
-    gridColumn(sizes: [.mobile: 12], [
-      div([
-        h2([`class`([Class.pf.type.responsiveTitle4])], ["Profile"]),
+private let profileRowView = View<AccountData> { data -> Node in
 
-        form([action(path(to: .account(.update(nil)))), method(.post)], [
-          label([`class`([labelClass])], ["Name"]),
-          input([
-            `class`([blockInputClass]),
-            name(ProfileData.CodingKeys.name.stringValue),
-            type(.text),
-            value(currentUser.name ?? ""),
-            ]),
+  let nameFields = [
+    label([`class`([labelClass])], ["Name"]),
+    input([
+      `class`([blockInputClass]),
+      name(ProfileData.CodingKeys.name.stringValue),
+      type(.text),
+      value(data.currentUser.name ?? ""),
+      ])
+  ]
 
-          label([`class`([labelClass])], ["Email"]),
-          input([
-            `class`([blockInputClass]),
-            name(ProfileData.CodingKeys.email.stringValue),
-            type(.email),
-            value(currentUser.email.rawValue)
-            ]),
+  let emailFields = [
+    label([`class`([labelClass])], ["Email"]),
+    input([
+      `class`([blockInputClass]),
+      name(ProfileData.CodingKeys.email.stringValue),
+      type(.email),
+      value(data.currentUser.email.rawValue)
+      ])
+  ]
 
-          ] + emailSettingCheckboxes.view(currentEmailSettings) + [
+  let extraInvoiceInfoFields = !data.isSubscriptionOwner ? [] : [
```

Comment:
the extra invoice info field is only included when the user is the subscription owner.